schema-kenerator-test/src/test/kotlin/io/github/smiley4/schemakenerator/test/ReflectionParser_SwaggerGenerator_Tests.kt

```diff
@@ -1202,22 +1210,24 @@ class ReflectionParser_SwaggerGenerator_Tests : FunSpec({
                                         "description": "String field description",
                                         "default": "A default String value",
                                         "exampleSetFlag": true,
-                                        "example": "An example of a String value"
+                                        "example": "An example of a String value",
+                                        "format": "text"
                                     },
                                     "intValue": {
                                         "types": ["integer"],
                                         "format": "int32",
                                         "description": "Int field description",
                                         "default": "1111",
                                         "exampleSetFlag": true,
-                                        "example": "2222"
+                                        "example": "2222",
```

Comment:
```suggestion
                                        "example": "2222"
```
the json parser doesn't like trailing commas and throws an exception here